apps/wpa_supplicant/src/p2p/p2p.c

```diff
@@ -19,19 +19,13 @@
 #include "p2p_i.h"
 #include "p2p.h"
 
-
 static void p2p_state_timeout(void *eloop_ctx, void *timeout_ctx);
 static void p2p_device_free(struct p2p_data *p2p, struct p2p_device *dev);
-static void p2p_process_presence_req(struct p2p_data *p2p, const u8 *da,
-				     const u8 *sa, const u8 *data, size_t len,
-				     int rx_freq);
-static void p2p_process_presence_resp(struct p2p_data *p2p, const u8 *da,
-				      const u8 *sa, const u8 *data,
-				      size_t len);
+static void p2p_process_presence_req(struct p2p_data *p2p, const u8 *da, const u8 *sa, const u8 *data, size_t len, int rx_freq);
```

Comment:
Please do not get me wrong. It is just an opinion of mine. Lines above were not written by me. :-)